src/components/GoodsList.tsx

```diff
@@ -0,0 +1,16 @@
+import { createRandomId } from '../utils/createRandomId';
+import { Good } from './Good';
+
+type GoodsListProps = {
+  goods: string[];
+};
+
+export const GoodsList: React.FC<GoodsListProps> = ({ goods }) => {
+  return (
+    <ul>
+      {goods.map(good => (
+        <Good goodName={good} key={createRandomId()} />
```

Comment:
```suggestion
        <Good goodName={good} key={good} />
```
you can just use good as a key